### DISCUSSION TITLE
TypeError: Cannot read properties of undefined (reading 'findUnique')

### DISCUSSION THREAD

**Discussion by Ladvace** [nextauthjs/next-auth#7162] [Help]
### Provider type

GitHub

### Environment

System:
    OS: macOS 13.3
    CPU: (10) arm64 Apple M1 Pro
    Memory: 368.08 MB / 32.00 GB
    Shell: 5.9 - /bin/zsh
  Binaries:
    Node: 16.14.2 - ~/.nvm/versions/node/v16.14.2/bin/node
    npm: 8.5.0 - ~/.nvm/versions/node/v16.14.2/bin/npm
  Browsers:
    Chrome: 111.0.5563.146
    Firefox: 111.0.1
    Safari: 16.4

### Reproduction URL

I used this template (https://github.com/OrJDev/create-jd-app) with trpc+tailwind+prisma+authjs

### Describe the issue

[auth][error][AdapterError]: Read more at https://errors.authjs.dev#adaptererror
[auth][cause]: TypeError: Cannot read properties of undefined (reading 'findUnique')
[auth][details]: {
  "provider": "github"
}

### How to reproduce

When I try signing in I get redirected to this URL and I got the error above

http://localhost:3000/api/auth/error?error=CallbackRouteError

### Expected behavior

successfully login

**Comment by Ladvace** [comment]
actually I tried another provider too (google) and I get the same error

**Reply by peterokwara** [reply]
I am just from having this error. I was using sveltekit, nextauth and prisma and Mongodb

```
[auth][details]: {}
[auth][error][AdapterError]: Read more at https://errors.authjs.dev#adaptererror
[auth][cause]: TypeError: Cannot read properties of undefined (reading 'findUnique')
```

I was using Atlas Mongodb what worked was

-  Ensuring my IP was allowed to connect to mlab MongoDB
-  Once it was allowed to connect, I could now run ` npx prisma db push`. Before I tried running it and I got the error

```
nvironment variables loaded from .env
Prisma schema loaded from prisma/schema.prisma
Datasource "db": MongoDB database "test" at "careergo-test-db.ch76mvq.mongodb.net"
Error: MongoDB error
Server selection timeout: No available servers. Topology: { Type: ReplicaSetNoPrimary, Servers: [ { Address: ac-uzkuali-shard-00-00.ch76mvq.mongodb.net:27017, Type: Unknown, Error: unexpected end of file }, { Address: ac-uzkuali-shard-00-01.ch76mvq.mongodb.net:27017, Type: Unknown, Error: unexpected end of file }, { Address: ac-uzkuali-shard-00-02.ch76mvq.mongodb.net:27017, Type: Unknown, Error: unexpected end of file }, ] }
   0: migration_core::commands::schema_push::Calculate `from`
             at migration-engine/core/src/commands/schema_push.rs:46
   1: migration_core::state::SchemaPush
             at migration-engine/core/src/state.rs:444
```

- But now it works

```
Applying the following changes:

[+] Collection `Account`
[+] Collection `Session`
[+] Collection `User`
[+] Collection `Payment`
[+] Collection `Log`
[+] Unique index `Account_provider_providerAccountId_key` on ({"provider":1,"providerAccountId":1})
[+] Unique index `Session_sessionToken_key` on ({"sessionToken":1})
[+] Unique index `User_email_key` on ({"email":1})

```

And now I can login with google!

**Reply by Rivalakmalll** [reply]
Hi i have same issue how did u fixed the eror to make user can login use google? im alrd tired anyhting 
but still get same eror
